gateway-ha/src/main/java/com/lyft/data/gateway/ha/handler/QueryIdCachingProxyHandler.java

```diff
@@ -145,6 +145,11 @@ public String rewriteTarget(HttpServletRequest request) {
         return null;
       }
     }
+    
```

Comment:
Is this a relic or good to leave in for debugging purposes?

Reply:
the newline?

Reply:
That comment didn't land well. The debug statement. I'm usually shocked to see any of them make it into production code